main.go

```diff
@@ -222,8 +217,10 @@ func (configs ConfigsModel) validate() error {
 		}
 	}
 
-	if err := validateTimeout(configs.TestTimeout); err != nil {
-		return fmt.Errorf("Issue with TestTimeout: %s", err)
+	if match, err := regexp.MatchString(timeoutPattern, configs.TestTimeout); err != nil {
```

Comment:
NOTE: should not we perform the normalize before the validation? will the regex match if there is a trailing `s`?